ui/components/app/menu-bar/account-options-menu.js

```diff
@@ -51,10 +60,14 @@ export default function AccountOptionsMenu({ anchorElement, onClose }) {
     },
   });
 
-  const keyring = useSelector(getCurrentKeyring);
-  const chainId = useSelector(getCurrentChainId);
-  const rpcPrefs = useSelector(getRpcPrefsForCurrentProvider);
-  const selectedIdentity = useSelector(getSelectedIdentity);
+  const customBlockExplorerLinkClickedEvent = useNewMetricEvent({
```

Comment:
Do we think this is the correct way to do this ? clicked view on etherscan vs block explorer? OR should we do 'clicked view on block explorer' and put the blockexplorer domain name in the event details?

Reply:
Fair point. I wasn't sure how we wanted this to fit into our segment schema / analytics. But you're probably right? Is there someone in specific that handles the segment analytics and might have thoughts/preferences here?

Reply:
@brad-decker ^

Reply:
Sorry about the delay. Lorenzo is the one to talk to, lets start a conversation with him in slack.

Reply:
@brad-decker made changes consolidate segment events.